app_legacy/Helpers/database/player-achievement.php

```diff
@@ -179,47 +181,6 @@ function insertAchievementUnlockIntoAwardedTable(string $user, int $achIDToAward
     return $dbResult !== false;
 }
 
-function getUsersRecentAwardedForGames(string $user, string $gameIDsCSV, int $numAchievements): array
-{
-    if (empty($gameIDsCSV)) {
-        return [];
-    }
-
-    $gameIDsArray = explode(',', $gameIDsCSV);
-
-    $gameIDs = [];
-    foreach ($gameIDsArray as $gameID) {
-        $gameID = (int) $gameID;
-        $gameIDs[] = $gameID;
-    }
-    $gameIDs = implode(',', $gameIDs);
-
-    $limit = ($numAchievements == 0) ? 5000 : $numAchievements;
```

Comment:
the 5000 hard-coded limit for the user page is no longer relevant. the unlocks will be managed per game by the `getUserAchievementUnlocksForGame` cache, and all unlocks for the N recent games will be returned.